libparsec/crates/protocol/schema/invited_cmds/invite_claimer_step.json5

```diff
@@ -0,0 +1,223 @@
+                "status": "not_ready"
+            },
+            // The following statuses do exist for the greeter, but not for the claimer
+            // Instead, in those cases, the claimer would get an HTTP 410 error, defined
+            // as `InvitationAlreadyUsedOrDeleted`.
+            // {
+            //     // The invitation has already been completed
+            //     "status": "invitation_completed"
+            // },
+            // {
+            //     // The invitation has been cancelled
+            //     "status": "invitation_cancelled"
+            // },
+            {
+                // The greeter has been revoked from the organization
+                "status": "greeter_revoked"
+            },
+            {
+                // The greeter is no longer part of the allowed greeters for this invitation
+                // An example of valid case for this error happens for a user invitation,
+                // if the profile of the greeter changes from ADMIN to NORMAL after
+                // `invite_claimer_start_greeting_attempt` was called by the claimer
+                "status": "greeter_not_allowed"
+            },
+            {
+                // The greeting attempt id doesn't correspond to any existing attempt
+                "status": "greeting_attempt_not_found"
+            },
+            {
+                // The author did not join the greeting attempt
+                // This should not happen, since joining is required to get the greeting attempt ID
+                "status": "greeting_attempt_not_joined"
+            },
+            {
+                // The greeting attempt has been cancelled
+                "status": "greeting_attempt_cancelled",
+                "fields": [
+                    {
+                        "name": "origin",
+                        "type": "GreeterOrClaimer"
+                    },
+                    {
+                        "name": "timestamp",
+                        "type": "DateTime"
+                    },
+                    {
+                        "name": "reason",
+                        "type": "CancelledGreetingAttemptReason"
+                    }
+                ]
+            },
+            {
+                // The submitted step is too advanced
+                // Every step before must have been completed by both peers
+                "status": "step_too_advanced"
+            },
+            {
+                // The submitted step somehow changed during polling
+                "status": "step_mismatch"
+            }
+        ],
+        "nested_types": [
+            {
+                // GreeterStep should be identical to the one in invite_greeter_step.json5
+                "name": "GreeterStep",
+                "discriminant_field": "step",
+                "variants": [
+                    {
+                        "name": "WaitPeer",
+                        "discriminant_value": "0",
```

Comment:
Use proper value instead of fake number ? the discriminant value is always a string, so better to use a value that is more human readable :thinking:

Reply:
See above.